### ISSUE TITLE
SNS Topic Subscription (Deleted) throwing errors 

### ISSUE THREAD

**Issue by AzySir** [cloudskiff/driftctl#601]
**Description**
Running driftctl against a statefile the only mention of SNS is in a policy. I've tried to do a .driftignore on the following resources but I'm hitting this debug. (I will run this with verbose mode and update this ticket) 

**Environment**
 - OS: Linux /bin/bash
 - driftctl version: 0.8.0

**How to reproduce**
Run driftctl

**Possible Solution**
Driftctl should accomodate for this scenario if possible. Possible warning rather than a failure?

**Additional context**

```

ERRO[0134] error reading SNS Topic Subscription (Deleted) attributes: InvalidParameter: Invalid parameter: SubscriptionArn Reason: An ARN must have at least 6 elements, not 1
--
status code: 400, request id: 3788ee62-553b-5248-a7e3-b75b65b4261a

Scanned resources (2875)
error reading SNS Topic Subscription (Deleted) attributes: InvalidParameter: Invalid parameter: SubscriptionArn Reason: An ARN must have at least 6 elements, not 1
status code: 400, request id: 3788ee62-553b-5248-a7e3-b75b65b4261a'

```

**Update**

I've ran this with the `LOG_LEVEL=DEBUG` flag and it is definitely failing on the SNS resources.

I've created this .driftignore file to ignore it temporarily but this does not seem to be working.

```
aws_sns_topic.*
aws_sns_topic_subscription.*
aws_sns_topic_subscription
aws_iam_policy.eit_permission_boundary
```

**Comment by moadibfr** [comment]
Seems to be a bug similar to what I already fixed in #345 but this time arn is "Deleted" instead of pending...

**Comment by eliecharra** [comment]
> Seems to be a bug similar to what I already fixed in #345 but this time arn is "Deleted" instead of pending...

Maybe we should ensure that we got a valid ARN to handle all cases when AWS decide not to return a real ARN there ?
